File: Pattern19.java
import java.util.Scanner;

//      *
//     * *
//    *   *
//   *     *
//  *       *
//   *     *
//    *   *
//     * *
//      *

public class Pattern19 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter the number:");
        int n = in.nextInt();
        int Corep;
        int spaceChange;
        for (int row = 1; row <= 2 * n - 1; row++)
    {
        Corep = row <= n ? row : 2 * n - row;
        spaceChange = row <= n ? n - row + 1 : (row - n) + 1;
        for (int k = 1; k <= spaceChange; k++)
        {
            System.out.print(" ");
        }
        for (int col = 1; col <= Corep; col++)
        {
            if (col == 1 || col == Corep)
            {
                System.out.print("* ");
            }
            else
            {
                System.out.print("  ");
            }
        }
        System.out.println();
    }
    }
}

File: Pattern19.py
    #      *
    #     * *
    #    *   *
    #   *     *
    #  *       *
    #   *     *
    #    *   *
    #     * *
    #      *

n = int(input("Enter the row number:"))
for row in range(1,2*n):
    Corep = row if row<=n else 2*n-row
    spaceChange = n-row+1 if row <= n else (row-n)+1
    for space in range(1,spaceChange+1):
        print(" ",end='')
    for col in range(1,Corep+1):
        if (col==1 or col==Corep):
            print("* ",end='')
        else:
            print("  ",end='')
    print()